src/commands/mobile_app/mod.rs

```diff
@@ -0,0 +1,48 @@
+use anyhow::Result;
+use clap::{ArgMatches, Command};
+
+use crate::utils::args::ArgExt;
+
+pub mod upload;
+
+macro_rules! each_subcommand {
+    ($mac:ident) => {
+        $mac!(upload);
+    };
+}
+
+pub fn make_command(mut command: Command) -> Command {
+    macro_rules! add_subcommand {
+        ($name:ident) => {{
+            command = command.subcommand(crate::commands::mobile_app::$name::make_command(
+                Command::new(stringify!($name).replace('_', "-")),
+            ));
+        }};
+    }
+
+    command = command
+        .about("(Experimental) Manage mobile apps.")
+        .subcommand_required(true)
+        .arg_required_else_help(true)
+        .org_arg()
+        .project_arg(true)
+        .release_arg()
```

Comment:
[question] is setting all of these going to be relevant for mobile app uploads?

Reply:
Project and org will be used. Release is TBD, but tentatively yes as that's the first spot we'll be fitting in. I'll go ahead and remove release and add it later should we need it.